src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,64 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+interface Props{
+  currentPage: number,
+  totalPages: number[],
+  onPageChange: (item: number) => void,
+  onChahgeByArrow: (back: string, currentPage: number) => void,
+}
```

Comment:
Please remove onChahgeByArrow and use only onPageChange
![image](https://github.com/mate-academy/react_pagination/assets/54936044/b84b59e3-248a-4ec1-b63e-5008a0e404ad)